src/app/staticanalysis/staticAnalysisRunner.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+var AstWalker = require('ethereum-remix').util.AstWalker
+var list = require('./modules/list')
+
+function staticAnalysisRunner () {
+}
+
+staticAnalysisRunner.prototype.run = function (ast, toRun, callback) {
+  var walker = new AstWalker()
+  for (var k in ast) {
+    walker.walk(ast[k].AST, {'*': function (node) {
```

Comment:
I think we could improve that in the future. Furthermore, there might be some things to check which need the AST as is, and not just as a callback on each node.